apps/demo/src/app/thank-you/pages/thank-you-view.component.html

```diff
@@ -0,0 +1,21 @@
+<daff-container size="md">
+  <div cart-container #CartContainer="CartContainer">
```

Comment:
This is fine for now, but just be aware that this isn't a "cart" this would be an "order".